Sources/XcodeGenKit/SettingPresets

```diff
@@ -0,0 +1 @@
+../../SettingPresets/
```

Comment:
What does this file do?

Reply:
This is symbolic link for SeetingPresets in root to detect it as bundle file for XcodeGenKit. Package.swift cannot find it if we don't have it in root of target source file here https://github.com/yonaskolb/XcodeGen/pull/1388/files#diff-f913940c58e8744a2af1c68b909bb6383e49007e6c5a12fb03104a9006ae677eR52

We also cannot set it as .copy("../../SettingPresets")